src/store/accounts/api.ts

```diff
@@ -0,0 +1,113 @@
+import { atomFamilyWithInfiniteQuery } from 'jotai-query-toolkit';
+import { atomFamilyWithQuery } from '@store/query';
+import { QueryRefreshRates } from '@common/constants';
+import { apiClientState } from '@store/common/api-clients';
+import type {
+  MempoolTransaction,
+  Transaction,
+  AccountDataResponse,
+} from '@stacks/stacks-blockchain-api-types';
+import type { AccountBalanceResponseBigNumber, AddressBalanceResponse, Keys } from './types';
+
+import BigNumber from 'bignumber.js';
+
+enum AccountClientKeys {
+  InfoClient = 'account/InfoClient',
+  BalancesClient = 'account/BalancesClient',
+  TransactionsClient = 'account/TransactionsClient',
+  MempoolTransactionsClient = 'account/MempoolTransactionsClient',
+}
+
+type PrincipalWithNetworkUrl = [principal: string, networkUrl: string];
+type PrincipalWithLimitNetworkUrl = [principal: string, limit: number, networkUrl: string];
```

Comment:
Can we use an object rather than a named array? When I later read this example, say: 

`set(accountMempoolTransactionsClient([address, 30, network.url]), { type: 'refetch' });`

It isn't clear what 30 means, without prior knowledge of this array structure. If this becomes:

```
set(accountMempoolTransactionsClient({ address, limit: 30, network: network.url }), { type: 'refetch' });
```

The next reader of this code doesn't have to reference back to see what it means.